net/stun/stun.go

```diff
@@ -223,7 +223,7 @@ func ParseResponse(b []byte) (tID TxID, addr netip.AddrPort, err error) {
 				return err
 			}
 			if len(a) == 16 {
-				addr6 = netip.AddrPortFrom(netip.AddrFrom16(*(*[16]byte)([]byte(a))), p)
+				addr6 = netip.AddrPortFrom(netip.AddrFrom16(*(*[16]byte)([]byte(a))).Unmap(), p)
```

Comment:
This works, but I think this is a little confusing that we might wind up calling something `addr6` when it is in fact an IPv4 address. Not sure that matters though.

Reply:
PTAL. Removed the two `6` variables. Not sure why they were there.